sfdx-project.schema.json

```diff
@@ -199,6 +203,11 @@
         }
       }
     },
+    "packageDirectory.functions": {
+      "title": "Functions To Package",
+      "type": "array",
```

Comment:
Does this only support string values in the array, like the example? If so, this should probably have
```
     "items": { "type": "string" }
```
in there.
Otherwise, I believe the json-schema default is that the array can contain anything (e.g. numbers, boolean, other arrays, objects, etc.)

Reply:
Good call.  I'll add.